lib/Zonemaster/CLI.pm

```diff
@@ -259,7 +259,7 @@ has 'nstimes' => (
     isa           => 'Bool',
     required      => 0,
     default       => 0,
-    documentation => __('At the end of a run, print a summary of the times the zone\'s name servers took to answer.'),
+    documentation => __( 'At the end of a run, print a summary of the times (in milliseconds) the zone\'s name servers took to answer.' ),
```

Comment:
This could be interpreted that it is presented at the end, which is not necessarily true for JSON output.